hoomd/hpmc/IntegratorHPMCMono.h

```diff
@@ -746,8 +746,8 @@ void IntegratorHPMCMono<Shape>::update(uint64_t timestep)
                 {
                 r_cut_patch = static_cast<OverlapReal>(m_patch->getRCut()) + static_cast<OverlapReal>(0.5) *
                     static_cast<OverlapReal>(m_patch->getAdditiveCutoff(typ_i));
-                r_cut_patch = static_cast<OverlapReal>(m_patch->getRCut()) +
-                    static_cast<OverlapReal>(0.5) * static_cast<OverlapReal>(m_patch->getAdditiveCutoff(typ_i));
+                r_cut_patch = static_cast<OverlapReal>(m_patch->getRCut()) + static_cast<OverlapReal>(0.5) *
+                    static_cast<OverlapReal>(m_patch->getAdditiveCutoff(typ_i));
```

Comment:
```suggestion
```
We only need to compute `r_cut_patch` once.

Reply:
I thought I had removed this repeated line in a previous commit, but I must have not actually removed it.

Ah, [this commit](https://github.com/glotzerlab/hoomd-blue/commit/3eca50ef8e29cd472f212e3947871545d7b36a84) is what I was thinking of, but I guess I just reformatted that line in that commit.